sdk/tables/azure-data-tables/azure/data/tables/_shared_access_signature.py

```diff
@@ -32,13 +32,13 @@ def __init__(self, account_name, account_key, x_ms_version=DEFAULT_X_MS_VERSION)
         """
         :param str account_name:
             The storage account name used to generate the shared access signatures.
-        :param str account_key:
+        :param AzureNamedKeyCredential account_key:
```

Comment:
hmm not sure this is needed for SAS generation. Or if we think it is - maybe we would also remove the `account_name` parameter, as that's also included in the credential.